tests/fixtures/example-com-well-known-webfinger.json

```diff
@@ -1 +1,18 @@
-{"headers":{"content-encoding":"gzip","accept-ranges":"bytes","age":"600634","cache-control":"max-age=604800","content-type":"text\/html; charset=UTF-8","date":"Fri, 03 May 2024 06:43:56 GMT","expires":"Fri, 10 May 2024 06:43:56 GMT","last-modified":"Fri, 26 Apr 2024 07:53:22 GMT","server":"ECAcc (dcd\/7D09)","vary":"Accept-Encoding","x-cache":"404-HIT","content-length":"648"},"body":"<!doctype html>\n<html>\n<head>\n    <title>Example Domain<\/title>\n\n    <meta charset=\"utf-8\" \/>\n    <meta http-equiv=\"Content-type\" content=\"text\/html; charset=utf-8\" \/>\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\" \/>\n    <style type=\"text\/css\">\n    body {\n        background-color: #f0f0f2;\n        margin: 0;\n        padding: 0;\n        font-family: -apple-system, system-ui, BlinkMacSystemFont, \"Segoe UI\", \"Open Sans\", \"Helvetica Neue\", Helvetica, Arial, sans-serif;\n        \n    }\n    div {\n        width: 600px;\n        margin: 5em auto;\n        padding: 2em;\n        background-color: #fdfdff;\n        border-radius: 0.5em;\n        box-shadow: 2px 3px 7px 2px rgba(0,0,0,0.02);\n    }\n    a:link, a:visited {\n        color: #38488f;\n        text-decoration: none;\n    }\n    @media (max-width: 700px) {\n        div {\n            margin: 0 auto;\n            width: auto;\n        }\n    }\n    <\/style>    \n<\/head>\n\n<body>\n<div>\n    <h1>Example Domain<\/h1>\n    <p>This domain is for use in illustrative examples in documents. You may use this\n    domain in literature without prior coordination or asking for permission.<\/p>\n    <p><a href=\"https:\/\/www.iana.org\/domains\/example\">More information...<\/a><\/p>\n<\/div>\n<\/body>\n<\/html>\n","response":{"code":404}}
```

Comment:
Can we exclude the test folder? there is no need to have the fixtures formatted properly. This only takes more space and we should keep the original file from the endpoints we test against to test as near as possible to the live environment.

Reply:
Fixed in [60f540a](https://github.com/Automattic/wordpress-activitypub/pull/1673/commits/60f540a57de17ec73e2e1b589bc25d843944961f).